frontend/packages/enduser-frontend/src/localization/app/fi.json

```diff
@@ -974,5 +974,9 @@
         "invalid-requested-start-date": "Aloitusta voi siirtää korkeintaan 14 päivää päätöksessä annetusta aloituspäivämäärästä. Aloituspäivämäärä ei voi olla aikaisempi kuin päätöksellä annettu aloituspäivämäärä."
       }
     }
+  },
+  "file-upload": {
```

Comment:
What's the plan for en/sv translations?

Reply:
There is no plan really (yet).

Reply:
Should we just Google Translate something reasonable? Suggestion:

- en:
  - loading: "Uploading..."
  - loaded: "Uploaded"
- sv:
  - loading: "Laddar upp..."
  - loaded: "Uppladdad"